Formula/kustomize.rb

```diff
@@ -34,8 +34,6 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/kustomize version")
```

Comment:
The version did not get populated correctly as `kustomize` and ` pluginator` are separate modules.